packages/plugins/plugin-inbox/src/components/Mailbox/Mailbox.stories.tsx

```diff
@@ -20,11 +55,56 @@ const DefaultStory = () => {
 
 export const Default = {};
 
+const WithCompanionStory = () => {
+  const space = useSpace();
+  const [mailbox] = useQuery(space, Filter.schema(MailboxType));
+  const state = useCapability(InboxCapabilities.MailboxState);
+
+  const mailboxData = useMemo(() => ({ subject: mailbox }), [mailbox]);
+  const message = mailbox && state[fullyQualifiedId(mailbox)];
+  const companionData = useMemo(() => ({ subject: message ?? 'message', companionTo: mailbox }), [message, mailbox]);
+
+  if (!space || !mailbox) {
+    return <div>Loading...</div>;
+  }
+
+  return (
+    <div className='grow grid grid-cols-2 overflow-hidden divide-x divide-divider'>
+      <Surface role='article' data={mailboxData} />
+      <Surface role='article' data={companionData} />
+    </div>
+  );
+};
+
+export const WithCompanion = {
+  render: WithCompanionStory,
+  decorators: [
+    withPluginManager({
+      plugins: [
+        ClientPlugin({
+          types: [MailboxType, MessageType],
+          onClientInitialized: async (_, client) => {
+            await client.halo.createIdentity();
+            await client.spaces.waitUntilReady();
+            await client.spaces.default.waitUntilReady();
+            await initializeMailbox(client.spaces.default);
+          },
+        }),
+        InboxPlugin(),
+        IntentPlugin(),
+        SettingsPlugin(),
+        SpacePlugin(),
+        ThemePlugin({ tx: defaultTx }),
```

Comment:
note: using theme plugin instead of `withTheme` decorator to automatically pull in all the plugins translations